build-system/tasks/process-github-issues.js

```diff
@@ -119,67 +124,118 @@ function updateGitHubIssues() {
           let milestoneTitle;
           let milestoneState;
           let hasPriority = false;
+          let hasCategory = false;
           let issueNewMilestone = MILESTONE_PENDING_TRIAGE;
-
-          // Get the title and state of the milestone
+          const assignee = issue['assignee'];
+          let assigneeName = '';
+          const issueLastUpdate = issue.updated_at;
+          let biweeklyUpdate = true;
+          let quartelyUpdate = true;
+          if (getLastUpdate(issueLastUpdate) > 89) {
+            quartelyUpdate = false;
+            biweeklyUpdate = false;
+          }
+          else if (getLastUpdate(issueLastUpdate) > 14) {
+            biweeklyUpdate = false;
+          }
+      // Get the assignee
+          if (assignee) {
+            assigneeName = '@' + assignee['login'];
+          }
+      // Get the title and state of the milestone
           if (milestone) {
-            milestoneTitle = milestone.title;
-            milestoneState = milestone.state;
+            milestoneTitle = milestone['title'];
+            milestoneState = milestone['state'];
           }
-          // Get the labels we want to check
```

Comment:
Why was the formatting in this block changed? Did you sync to master before editing this file? Please undo all the unnecessary changes in this file.

Reply:
done